### ISSUE TITLE
eem_read not creating eemlist class objects

### ISSUE THREAD

**Issue by JMahaux** [PMassicotte/eemR#73]
Hi Philippe,

I'm having an issue with the eem_read function in that it's not creating objects of class eemlist. I have also tried using the "eem" function to construct the object but keep getting an error saying R can't find the function "eem" (along with some of the other eemR functions, like "summary.eemlist"). 
I'm importing data from an aqualog and am able to create a list object using eem_read, but not an eemlist, which seems to be giving me trouble when I try to do blank subtractions and raman normalisation since I believe the function is not recognizing my inputs as valid - using eem_remove_blank with the eem and blank lists returns "Error in if (is.na(blank)) { : the condition has length > 1"

The package is loaded correctly and the function shows up when I use "?eem" so I'm at a bit of a loss about what's going wrong.
I'm relatively new to R so it's entirely possible I've missed a crucial step somewhere, but any help here would be appreciated!

Thanks!

**Comment by PMassicotte** [comment]
Hi.

Are you able to run the examples provided in the package help?

**Comment by JMahaux** [comment]
Yes, it seems to work with the example data so it's likely an issue with my
input formatting. I'm trying to get the various indices (BIX, HIX, FI) with
data from many days' runs and currently have a folder with all of the
sample EEMs and a folder with a matched blank file (BEM.dat from the
aqualog) for each sample. Should these instead be the daily .blank files?

**Comment by PMassicotte** [comment]
If `eem_read()` works on your data, it should return an object of class `eemlist`. Your data should be organized (not mandatory, but will help for automatic correction) as this:

https://github.com/PMassicotte/eemR/tree/main/inst/extdata/cary

Is it how you have it organized?

**Comment by PMassicotte** [comment]
Maybe some information for you here: https://pmassicotte.github.io/eemR/articles/introduction.html
